extra_foam/pipeline/data_model.py

```diff
@@ -188,12 +188,15 @@ def __init__(self):
 
 class AzimuthalIntegrationData(DataItem):
     """Azimuthal integration data item."""
-    __slots__ = ['x', 'y', 'fom', 'q_map', 'peaks']
+    __slots__ = ['q_map', 'peaks', 'max_peak', 'max_peak_q', 'center_of_mass']
```

Comment:
We're still using `ai.x`, `ai.y`, and `ai.fom`, so I guess these shouldn't be removed

Reply:
They weren't fully removed, they're originally declared in `DataItem` and this subclass re-declared them. That isn't recommended though so only the duplicate slots from the subclass are removed.